crowdin.template.yml

```diff
@@ -0,0 +1,7 @@
+project_id: ${CROWDIN_PROJECT_ID}
```

Comment:
This is a template that will be used by the `generatedCrowdinConfig.js` script to generate the `crowdin.yml` file need to upload and download content from crowdin. 

The reason I am doing it this way is so that I can use environment variables to set `project_id` and `api_token` in `crowdin.yml` instead of hard-coding it in the file.